### DISCUSSION TITLE
How to config RBAC for gitlab sso user?

### DISCUSSION THREAD

**Discussion by JulianZhang** [argoproj/argo-cd#6613] [Q&A] [answered]
Dear:

  I had config the gitlab sso with dex, and the user login is work now.
  And I can add gitlab group to the project roles. So the user can see all the applications in the project.

  But the user can't get repo and cluster info ,so it can't create new application.

  I had set RBAC for local user to get the repo/cluster before. 
  I had try many way for the gitlab sso user but none of it worked.

  Is there any one can give me a workable example for gitlab user RBAC config?

 Thanks very much

**Comment by iam-veeramalla** [comment, marked as answer]
Hi @JulianZhang

Does the GitLab User show up the Argo CD user Profile section after your Login ? Can you please paste a screenshot of your Argo CD User Profile page here ?

I hope you already looked into the Argo CD RBAC examples ?
https://argo-cd.readthedocs.io/en/stable/operator-manual/rbac/

**Reply by JulianZhang** [reply]
dear, I had fix the problem.

I think I had misunderstand with the github example. I was think the org level is need for gitlab too. 

But I find just use gitlab group name will make the RBAC setting worked.

**Reply by iam-veeramalla** [reply]
That's Great , Thank you !!
